content/chat/rooms/reactions.textile

```diff
@@ -0,0 +1,82 @@
+---
+title: Room reactions
+meta_description: "Enable users to react at the room level to what is happening in your application."
+product: chat
+languages:
+  - javascript
+---
+
+Users can react to what is happening within a room to show their sentiment as to what is happening. For example, agreeing with the content in a livestream using a thumbs up, or sending a heart when their team scores in a sports game.
+
+These reactions are ephemeral and not stored by Ably. The intention being that they show the overall room sentiment at a point in time.
```

Comment:
I would move this inside the note and say:

Room reactions are currently ephemeral and not stored or aggregated by Ably.

Reply:
i.e. being careful about not promising a future update but at the same time, giving a taste of what we know is missing atm.

Reply:
Updated wording, but don't think they make sense to be in the note with how to enable the feature.